src/Microsoft.Identity.Client/AppConfig/AadAuthorityAudience.cs

```diff
@@ -45,10 +45,12 @@ namespace Microsoft.Identity.Client.AppConfig
     */
 
     /// <summary>
+    /// Specifies what Microsoft accounts can be signed-in in a given application.
```

Comment:
suggestion: Specifies which Microsoft accounts can be used for sign-in with a given application